src/main/java/mate/controller/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final String SESSION_ATTRIBUTE_NAME = "driver_id";
```

Comment:
```suggestion
    private static final String SESSION_DRIVER_ID = "driver_id";
```
there can be a lot of attribute names in future